src/components/Operations/BookmarkMotion/bookmark--motion.module.scss

```diff
@@ -3,7 +3,7 @@
     transform: translatey(0px);
   }
   10% {
-    transform: translatey(-4px);
+    transform: translatey(4px);
```

Comment:
This definitely still works with lowercase `y`, but best practice for CSS would be with capital `translateY`.  Not a blocker for merging, just FYI for future 😄

Reply:
Resolving this convo since this is just a comment, no change required right now